content/docs/machine-learning/quickstart.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 2
+title: Get Started with Kubeflow
+description: Learn how to get started with Civo's Kubeflow via an example.
+---
+
+<head>
+  <title>Kubeflow quickstart | Civo Documentation</title>
+</head>
+
+In this quickstart mainly focused towards Pipelines and Notebooks, we take a look at a simple example to train and run a model using Kubeflow Pipelines and Notebooks.
+
+## Before you start
+
+Make sure you can log into your Kubeflow cluster by following the instructions at [Logging into your cluster](kubeflow-dashboard.md#logging-into-your-cluster).
+
+Make sure you can create a notebook and log into the notebook by following the instructions at [Creating a Kubeflow Notebook](creating-a-new-kubeflow-notebook.md).
+
+## Run a pipeline
+
+We will first compile a pipeline DSL that will train a model using the MNIST dataset. Notice that this pipeline mainly has the following coponents which are run: hyperparameter tuning with Katib, creating a new volume for training, run a training job and finally serve the model using KServe.
+
+Now [create a new notebook instance](creating-a-new-kubeflow-notebook.md) and run the following commands:
+
+```bash
+git clone 
+cd 
+pip install -r requirements.txt
+python mnist-example.py
+```
+
+This produces for us a `mnist-example.yaml` file that we will use to run our pipeline.
+
+We now use [Pipelines](kubeflow-dashboard.md/) and use the pipeline configuration we generated in the previous step to define the pipeline. Once you do so, you should create a new [experiment](kubeflow-dashboard.md/) and then trigger a run for the pipline by going to the Run tab, clicking on the "create a run" button and choosing the pipeline you just created.
```

Comment:
typo "pipline" -> "pipeline"

Worth having a screenshot?

Reply:
got it

Reply:
Do we want some other screenshots in the docs: like pipeline being run successfully, or output from the endpoint, and so on?

Reply:
Yeah, I think so. With text output as code blocks rather than screenshots of terminals if that's what the output is